### PR TITLE
A few tweaks for Python 3

### PR DESCRIPTION
**Pull request by msabramo**
Working on getting MySQLdb to work in Python 3.

Current status:

```
$ tox -e py33
...
Ran 69 tests in 6.115s

FAILED (errors=22)
...
```

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[tox]
+envlist = py25, py26, py27, py33
```
Comment (methane) [inline tox.ini]: This pull request should not work with py25.

File: MySQLdb/cursors.py [hunk 2]
```diff
@@ -14,6 +14,7 @@
     ListType = list
     TupleType = tuple
     UnicodeType = str
+    unicode = str
```
Comment (methane) [inline MySQLdb/cursors.py]: remove UnicodeType instead.

File: setup_posix.py [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import os, sys
-from ConfigParser import SafeConfigParser
+import six.moves
+SafeConfigParser = six.moves.configparser.ConfigParser
```
Comment (methane) [inline setup_posix.py]: Do you need six for only this?

File: MySQLdb/connections.py [hunk 4]
```diff
@@ -229,8 +232,8 @@ def string_decoder(s):
             self.converter[FIELD_TYPE.VARCHAR].append((None, string_decoder))
             self.converter[FIELD_TYPE.BLOB].append((None, string_decoder))
 
-        self.encoders[types.StringType] = string_literal
-        self.encoders[types.UnicodeType] = unicode_literal
+        self.encoders[bytes] = string_literal
+        self.encoders[str] = unicode_literal
```
Comment (methane) [inline MySQLdb/connections.py]: Add `unicode = str` like cursor and use it.

**Comment by vstinner** [comment]
What is the status of this issue? I would like Python 3 support for MySQL-python.

File: MySQLdb/cursors.py (hunk 2 above)
Comment (msabramo) [inline MySQLdb/cursors.py]: Why remove `UnicodeType`?

Comment (farcepest) [inline MySQLdb/cursors.py]: In Python 2.x, unicode is UnicodeType.  In Python 3.x, unicode (and UnicodeType) are set to str. UnicodeType is used at line 299. unicode is used in a variety of places.

**Comment by msabramo** [comment]
Note: https://github.com/farcepest/MySQLdb1/pull/48 is extremely similar.

**Comment by msabramo** [comment]
Check out #62. @methane has taken what I have here and added a bunch of great stuff, including modifying the C code to not segfault on Python 3.

**Comment by msabramo** [comment]
Ping

**Comment by msabramo** [comment]
This project hasn't had a commit in over a year, so I'll just close this.

**Pull request closed without merging**